tests/CMakeLists.txt

```diff
@@ -113,21 +90,18 @@ message (STATUS "Tests enabled, test executable will be built as tests/openshot-
 
 add_executable(openshot-test
   tests.cpp
-  ${OPENSHOT_TEST_FILES} 
-  ${OPENSHOT_CV_TEST_FILES}
-  )
+  ${OPENSHOT_TEST_FILES}
+)
 
 # Link libraries to the new executable
 target_link_libraries(openshot-test
   openshot
   ${UnitTest++_LIBRARIES}
-  ${OPENSHOT_CV_LIBRARIES}
-  )
```

Comment:
Again, the use of CMake targets instead of variables means that it takes care of propagating these dependencies automatically.